### PR TITLE
IN-5131: Added Experience integrations

### PR DESCRIPTION
**Pull request by adnanrahic**
Added SPA+static+SSR integrations page.

### CODE REVIEW

File: docs/experience/integrations.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
+title: Sematext Experience Integrations
+descriptions: Sematext integrations with ready to use monitoring agents and log shippers for infrastructure and container monitoring, log management and analytics, alerting, chatops, and more. Our Cloud and on-premises platform exposes the Elasticsearch API and syslog receivers with built in anomaly detection, data analysis and visualization tools and services
```
Comment (otisg) [inline docs/experience/integrations.md]: If description matters we should update it to be more about User Experience / Real User Monitoring

File: docs/experience/integrations.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+				</div>
+			</div>
+		</a>
+	</div>
+</div>
+
+All of the integrations require adding and [configuring the Experience script](./getting-started). However, single-page applications require one more configuration step to register route changes.
+
+## Angular
+The Angular SPA integrations supports both Angular.js and Angular 2+.
```
Comment (otisg) [inline docs/experience/integrations.md]: integrations => integration

File: docs/experience/integrations.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+	</div>
+</div>
+
+All of the integrations require adding and [configuring the Experience script](./getting-started). However, single-page applications require one more configuration step to register route changes.
+
+## Angular
+The Angular SPA integrations supports both Angular.js and Angular 2+. 
+
+### Angular 2+
+If your website uses newer Angular.js you should add a call to routeChange whenever the route changes in your webapp. This can be done in your top-level component where Router is defined by adding the ngOnInit function and subscribing to the Router events as shown in the following example.
```
Comment (otisg) [inline docs/experience/integrations.md]: website or webapp or app or frontend app or ...?

routeChange => `routeChange` ?
Router => `Router`?

File: docs/experience/integrations.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+      if (event instanceof NavigationStart) {
+        strum('routeChange', event.url);
+      }
+    });
+  }
+}
+```
+
+### Angular.js
+If your website uses Angular.js 1.x you should add a call to routeChange whenever the route changes in your webapp. To do this you'll need to adjust your client side view inside Angular:
```
Comment (otisg) [inline docs/experience/integrations.md]: website or webapp or app or frontend app or ...?

routeChange => `routeChange` ?

File: docs/experience/integrations.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+  return (
+    <Router history={history}>
+      ...
+    </Router>
+  );
+}
+```
+
+## Vue.js 
+The Vue.js integration works by wathing for calls to the `routeChange` function where you have the `router-view` defined.
```
Comment (otisg) [inline docs/experience/integrations.md]: wathing => watching

File: docs/experience/integrations.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+  <div id="app">
+    <router-view/>
+  </div>
+</template>
+<script>
+  export default {
+    name: 'app',
+    watch: {
+      $route() {
+        // eslint-disable-next-line
```
Comment (otisg) [inline docs/experience/integrations.md]: Need this comment?

File: docs/experience/integrations.md [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+The Ember integrations uses the `reopen` function to configure a function to trigger every time your application changes routes. This event is called `didTransition`.
+
+```js
+
+import EmberRouter from '@ember/routing/router';
+import { on } from '@ember/object/evented';
+
+EmberRouter.reopen({
+  doInformAboutRouteChange: on('didTransition', function() {
+    // eslint-disable-next-line
```
Comment (otisg) [inline docs/experience/integrations.md]: Need this comment?

Comment (adnanrahic) [reply docs/experience/integrations.md]: Removed

File: docs/experience/integrations.md (hunk 6 above)
Comment (adnanrahic) [reply docs/experience/integrations.md]: Removed

File: docs/experience/integrations.md (hunk 5 above)
Comment (adnanrahic) [reply docs/experience/integrations.md]: Fixed

File: docs/experience/integrations.md (hunk 4 above)
Comment (adnanrahic) [reply docs/experience/integrations.md]: Webapp. 
Fixed.

File: docs/experience/integrations.md (hunk 3 above)
Comment (adnanrahic) [reply docs/experience/integrations.md]: Webapp.
Fixed.

File: docs/experience/integrations.md (hunk 2 above)
Comment (adnanrahic) [reply docs/experience/integrations.md]: Fixed.

File: docs/experience/integrations.md (hunk 1 above)
Comment (adnanrahic) [reply docs/experience/integrations.md]: I've added a more suiting description.

File: docs/experience/integrations.md (hunk 3 above)
Comment (otisg) [reply docs/experience/integrations.md]: ngOnInit => `ngOnInit` ?

Comment (adnanrahic) [reply docs/experience/integrations.md]: Yep, I added that fix as part of the last commit as well. :sunglasses:

**Pull request merged**